### DISCUSSION TITLE
Deleting a user via REST not working

### DISCUSSION THREAD

**Discussion by PoeHaH** [supabase/supabase#7781] [Questions] [answered]
I'm trying to make a DELETE call to `{url}/admin/users/{user_id} ` but it returns "no Route matched with those values". My apikey is correctly set to the service_role. 

Is this not the correct call to delete a user? Or is there a way I can do it in the dashboard?

**Comment by GaryAustin1** [comment]
That general format looks correct.
From gotrue-js:
```
async deleteUser(
    uid: string
  ): Promise<{ user: User | null; data: User | null; error: ApiError | null }> {
    try {
      const data: any = await remove(
        this.fetch,
        `${this.url}/admin/users/${uid}`,
        {},
        {
          headers: this.headers,
        }
      )
```
Dashboard delete is done in the auth section users tab:
![image](https://user-images.githubusercontent.com/54564956/179758395-436aabcc-c534-487b-8f0e-3ef0042baaa1.png)

**Comment by silentworks** [comment, marked as answer]
What tool are you using to make this API call? and can you show us a screenshot of said tool with the query please?

**Reply by PoeHaH** [reply]
I'm testing it in Postman. I use the url `/admin/users/{id}` as I found in the javascript SDK (because it's not documented anywhere). I make sure I have a Bearer token and the apikey in the headers. The api key is the `service_role`. 

![image](https://user-images.githubusercontent.com/5845313/180475681-19073494-99ca-4245-87a3-e3be94476fab.png)

So it looks correct to me, but the response is "no route matching".

**Reply by GaryAustin1** [reply]
So the URL you are using is wrong one and should be the gotrue (auth) url.
https://xxxxxxxxxxxxxxxpbq.supabase.co/auth/v1/admin/users/2ae5a658-146e-499c-b428-d2efc26fa1a8
Is the URL gotrue.js generates for the DELETE operation.

**Reply by PoeHaH** [reply]
Understood, thanks! Might need to be documented somewhere :)

**Reply by GaryAustin1** [reply]
One of the best resources for auth and database api formatting with http calls is the API tab in the dashboard.  Then select Bash.  
![FAD0E15C-6CBE-4A17-94BB-6B39A1512468](https://user-images.githubusercontent.com/54564956/180604255-1327e8c9-6033-4880-9ce0-5db36c8f9540.jpeg)

**Comment by J0** [comment]
@GaryAustin1 is right. The `https://xxxxxxxxxxxxxxxpbq.supabase.co/auth/v1/admin/users/<user_id>` uri should be used to delete a user
